src/GraphQL.FluentValidation/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FluentValidation;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for <see cref="IServiceCollection"/> to add fluent validators types.
+    /// </summary>
+    public static class ServiceCollectionExtensions
```

Comment:
Hmm, I wonder if the default here should be Transient, which aligns with the default lifetime in FluentValidation.
https://github.com/FluentValidation/FluentValidation/blob/de1400928b9517bedf3cb8936b72a3d0334d0180/src/FluentValidation.DependencyInjectionExtensions/ServiceCollectionExtensions.cs#L33

Reply:
Let's do it easier - just use `FluentValidation.DependencyInjectionExtensions`.

Reply:
Yep, nice one.